### DISCUSSION TITLE
Is there an alternative to vue-tree-select?

### DISCUSSION THREAD

**Discussion by oussama-he** [quasarframework/quasar#11112] [General - Components / Directives / etc] [answered]
I've tried to use the [vue-tree-select](https://vue-treeselect.js.org/) package in my project, but it seems is no longer maintained (last commit on Nov 8, 2019).

I would like to know if there is a component or an alternative to this package in quasar.

Thank you in advance.

**Comment by hawkeye64** [comment, marked as answer]
QTree
